Assets/Scripts/StarComponent.cs

```diff
@@ -11,9 +11,16 @@ public class StarComponent : MonoBehaviour, IPointerDownHandler, IPointerExitHan
     private Vector3 initialScale;
     private float scaleFactor = 1.5f;
 
-    public void Init(ConstellationsController constellationsController)
+    public void Init(ConstellationsController controller, Star star, float maxMagnitude, float magnitudeScale, float radius)
     {
-        constellationsController = constellationsController;
+        constellationsController = controller;
+        starData = star;
+        transform.position = star.CalculateEquitorialPosition(radius);
+
+        var magScaleValue = ((star.Mag * -1) + maxMagnitude + 1) * magnitudeScale;
+        Vector3 magScale = transform.localScale * magScaleValue;
+        transform.localScale = magScale;
+        transform.LookAt(constellationsController.transform);

```

Comment:
We lost the calculation above, but I wonder if we should eventually yank these and simply ensure that `UpdateOnSceneLoad` in `DataController` is called every time a scene is loaded.  Then the localScale would always be set there.